Add def methods without parentheses

Just wondering if it's possible to include also methods without parentheses, like:

```rb
def some_method parameter_1 parameter_2
end
```

I was trying to add a test, but I'm getting this:

```
/usr/lib/electron/electron --debugBrkPluginHost=31500 --debugId=d56f2c9b-c651-4044-8458-069c1e343f5f /home/pablo/code/vscode/vscode-yard/src/test/project --extensionDevelopmentPath=/home/pablo/code/vscode/vscode-yard --extensionTestsPath=/home/pablo/code/vscode/vscode-yard/out/test 
Error launching app
Unable to find Electron app at /home/pablo/code/vscode/vscode-yard/src/test/project
Cannot find module '/home/pablo/code/vscode/vscode-yard/src/test/project'
```

What I did:
1. Clone the repo
2. `npm install`
3. Open the folder on VS Code
4. Debug sidebar
5. Extension tests
6. Little green arrow

Nevertheless, I'm not that familiarized with javascript/typescript, but I was going to give it a go.

--- a/src/parsers/definitions/method.ts
+++ b/src/parsers/definitions/method.ts
@@ -15,7 +15,7 @@ export default class MethodDef implements IBaseParser {
       /(?:def)\s+/, // def
       /(?:self)?\.?/, // self.
       /([a-zA-Z_\u3000-\u303f\u3040-\u309f\u30a0-\u30ff\uff00-\uff9f\u4e00-\u9faf\u3400-\u4dbf][a-zA-Z_0-9\u3000-\u303f\u3040-\u309f\u30a0-\u30ff\uff00-\uff9f\u4e00-\u9faf\u3400-\u4dbf]+[!?=]?|\+|-|\*|\*\*|\/|%|&|\^|>>|<<|==|!=|===|=~|!~|<=>|<|<=|>|>=|\[\]=|\[\]|\+@|-@|!@|~@)?/, // method name
-      /(\(.*\))?/, // method parameters
+      /(\(.*\)|[^;].*)?/, // method parameters
     ].map((r) => r.source).join(""),
   );
   // tslint:enable:max-line-length

--- a/src/test/extension.test.ts
+++ b/src/test/extension.test.ts
@@ -70,6 +70,7 @@ suite("Default Documenter Tests", () => {
     "method/single_line_parentheses",
     "method/initializer",
     "method/class_single_param",
+    "method/without_parentheses",
     "method/naming/setter",
     "method/naming/bang",
     "method/naming/boolean",

--- a/src/test/project/method/without_parentheses.rb
+++ b/src/test/project/method/without_parentheses.rb
@@ -1,0 +1,2 @@
+def foo bar, baz = 1, qux: 2
+end

--- a/src/test/project/method/without_parentheses_expected.rb
+++ b/src/test/project/method/without_parentheses_expected.rb
@@ -1,0 +1,11 @@
+#
+# <Description>
+#
+# @param [<Type>] bar <description>
+# @param [<Type>] baz <description>
+# @param [<Type>] qux <description>
+#
+# @return [<Type>] <description>
+#
+def foo bar, baz = 1, qux: 2
+end

